src/dictionary/ui/display/wordInformationDisplay.css

```diff
@@ -0,0 +1,6 @@
+.linkContainer:hover {
+  cursor: pointer;
+}
+.markdown {
```

Comment:
What's this for? There is no `markdown` class in the code.

Use BEM naming conventions.